packages/@aws-cdk/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -14,8 +14,11 @@ class EksClusterStack extends TestStack {
       assumedBy: new iam.AccountRootPrincipal(),
     });
 
+    const vpc = new ec2.Vpc(this, 'Vpc',  { maxAzs: 3, natGateways: 1});
```

Comment:
* add a comment explaining why this is needed

* style:

```suggestion
    const vpc = new ec2.Vpc(this, 'Vpc',  { maxAzs: 3, natGateways: 1 });
```